Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollViewportUiCluster.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Math/Transform.h>
+#include <AzToolsFramework/ViewportUi/ViewportUiRequestBus.h>
+#include <Editor/Plugins/Ragdoll/RagdollManipulators.h>
+
+namespace EMotionFX
+{
+    class Actor;
+    class Node;
+    class Transform;
+
+    struct RagdollViewportUiClusterData
+    {
+    };
+
+    class RagdollViewportUiCluster
```

Comment:
nit: API comment could be useful for the public type

Reply:
Added.